packages/core/components/ViewportControls/default-viewports.ts

```diff
@@ -0,0 +1,7 @@
+import { Viewports } from "../../types/Viewports";
+
+export const defaultViewports: Required<Viewports> = [
+  { width: 360, height: "auto", icon: "Smartphone", label: "Small" },
+  { width: 768, height: "auto", icon: "Tablet", label: "Medium" },
+  { width: 1280, height: "auto", icon: "Monitor", label: "Large" },
+];
```

Comment:
@monospaced do these seem like sensible defaults to you?

Reply:
Yes, I did consider that in review, and I think they do ✅